app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/JenkinsModal.tsx

```diff
@@ -1,26 +1,32 @@
-import Button, { LoadingButton } from '@atlaskit/button';
+import Button, { LoadingButton, Appearance } from '@atlaskit/button';
 import React from 'react';
 import Modal, {
-	Appearance,
 	KeyboardOrMouseEvent,
 	ModalBody,
 	ModalFooter,
 	ModalHeader,
 	ModalTitle,
-	ModalTransition
+	ModalTransition,
+	Appearance as ModalAppearance
```

Comment:
Updated this so we were using the correct type for the modal appearance. Previously we were pulling in the type from the Button component which does not give the correct types.